src/hotspot/share/gc/shenandoah/shenandoahMarkBitMap.hpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2018, 2019, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2018, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Probably should be 2018, 2019, 2023

Reply:
This and the other ones are correct, it's `$FirstChange, $LastChange, `